internal/service/backupcompliancepolicy/resource_backup_compliance_policy_test.go

```diff
@@ -41,7 +42,7 @@ func TestAccGenericBackupRSBackupCompliancePolicy_basic(t *testing.T) {
 	})
 }
 
-func TestAccGenericBackupRSBackupCompliancePolicy_withFirstLastName(t *testing.T) {
+func TestAccGenericBackupRSBackupCompliancePolicy_update(t *testing.T) {
```

Comment:
would it be possible to have a test that captures the behaviour:
1. have a TF with consistent backup and backup policy data
2. try to change backup policy
3. catch the validation error

Reply:
added test to catch error